### PR TITLE
Add new parser SystemdRpcbindSocketConf

### PR DESCRIPTION
**Pull request by msekletar**
* Add new data source implemented by simple_command()

* Add new parser which produces the parsed content of rpcbind.socket unit file

### CODE REVIEW

File: insights/parsers/systemd/config.py [hunk 1]
```diff
@@ -170,6 +170,37 @@ class SystemdLogindConf(SystemdConf):
...
+        ListenDatagram=0.0.0.0:111
+        ListenStream=[::]:111
+        ListenDatagram=[::]:111
+
+        [Install]
+        WantedBy=sockets.target
+
+    Example:
+        >>> rpcbind_socket["Socket"]["ListenStream"]
+        ['/var/run/rpcbind.sock', '0.0.0.0:111', '[::]:111']
```
Comment (wushiqinlou) [inline insights/parsers/systemd/config.py]: it should be ['/run/rpcbind.sock', '0.0.0.0:111', '[::]:111']

**Comment by wushiqinlou** [comment]
@msekletar - Please add test data and doc_example item in tests/test_systemd_config.py by referring to other parse.

**Comment by msekletar** [comment]
@wushiqinlou Thanks for the review! I have fixed the comment and added tests. PTAL

**Comment by wushiqinlou** [comment]
@msekletar - The logic is good to me, however, there are server flake8 issues. Please run "flake8 <your modified file>" to find out the issues and fix them.

**Comment by msekletar** [comment]
@xiangce thanks! I will start working on the remaining points.

**Comment by msekletar** [comment]
@xiangce I've updated my branch. PTAL.

I've also opened PR against insights-core-assets repo,

https://github.com/RedHatInsights/insights-core-assets/pull/204

**Pull request merged**